mixitup/index.html

```diff
@@ -663,142 +77,29 @@ <h3 class="nameOnModal">Priyanka Oberoi</h3>
                     <div class="row">
                         <div class="col-lg-4 col-md-4 col-sm-4">
                             
-                            <img class="img-responsive img-centered img-circle" src="BioPics/Burton.jpg" alt="Burton Reist">
+                            <img class="img-responsive img-centered img-circle" src="BioPics/{{picture_file}}" alt="{{first_name}} {{last_name}}">
                             
                         </div>
                         <div class="col-lg-7  col-md-7 modalTextStyles" style="text-align:left; color: black">
 
                                 <!-- Bio Details Go Here -->
 
-                                <h3 class="nameOnModal">Burton Reist</h3>
-                                <h4>Director of External Affairs</h4>
+                                <h3 class="nameOnModal">{{first_name}} {{last_name}}</h3>
```

Comment:
I noticed this line because the class name was mixed case. However, I wonder if adding another class is the best way to do this... The styling could probably be applied by just adding a CSS block for `.modal-content > h3`.